base_layer/wallet_ffi/src/lib.rs

```diff
@@ -5205,11 +5205,11 @@ pub unsafe extern "C" fn wallet_import_utxo(
         return 0;
     }
 
-    if features.is_null() {
-        error = LibWalletError::from(InterfaceError::NullError("features".to_string())).code;
-        ptr::swap(error_out, &mut error as *mut c_int);
-        return 0;
-    }
+    let features = if features.is_null() {
+        OutputFeatures::default()
```

Comment:
@hansieodendaal is this alright? Will a byte of 0 be treated correctly?

Reply:
The features and metadata signature must be consistent. Default features are only allowed for `V0`